README.md

```diff
@@ -1,8 +1,65 @@
 # Moorhuhn
 
-Unity Version: 2021.3.1f1 (LTS)
+In this game you have to shot the chicken with the right answer.  
 
-## Moorhuhn Minigame target functions
+## Development
+
+### Getting started
+
+Install the [Unity Version 2021.3.1f1 (LTS)](https://gamifyit-docs.readthedocs.io/en/latest/dev-manuals/languages/unity/version.html)
+
+Clone the repository  
+```sh
+git clone https://github.com/Gamify-IT/moorhuhn.git
+```
+
+
+### Build
+
+Build the project like discribed in [this manual](https://gamifyit-docs.readthedocs.io/en/latest/dev-manuals/languages/unity/build-unity-project.html).
+
+Build the Docker-Container
+```sh
+docker build -t moorhuhn-dev
+```
+And run it at port 8000 with
+```sh
+docker run -d -p 8000:80 --name moorhuhn-dev moorhuhn-dev
+```
+
+To monitor, stop and remove the container you can use the following commands:
+```sh
+docker ps -a -f name=moorhuhn-dev
+```
+```sh
+docker stop moorhuhn-dev
+```
+```sh
+docker rm moorhuhn-dev
+```
+
+## User manual
+
+Run the docker container with the following command at port 8000:
+```sh
+docker run -d -p 8000:80 --name moorhuhn ghcr.io/gamify-it/moorhuhn:latest
+```
+Now you can access it at [http://localhost:8000](http://localhost:8000).  
+To access it externally replace localhost with your IP.  
+
+To monitor the container you can use the following command:
+```sh
+docker ps -a -f name=moorhuhn
+```
+To stop the container you can use the following command:
+```sh
+docker stop moorhuhn
+```
+To remove the container you can use the following command:
+```sh
+docker rm moorhuhn
+```
```

Comment:
I wonder if we want to simply set an `image` variable and then referring to the specific docs README,
or if we always want to copy the docs READMEs.
I'm fine with either approach.

Reply:
What I meant is using something like "Use as described in $URL for `IMAGE_NAME=moorhuhn-dev`" instead of these paragraphs that simply copy that file.

But as I said, I'm fine with both approaches.

Reply:
Ok now I got it. 
Yes the monitor, stop and remove command are always the same, but I would choose the copy cause of better readability